frontend/src/views/WheresMyGene/components/Main/index.tsx

```diff
@@ -347,6 +352,13 @@ export default function WheresMyGene(): JSX.Element {
             />
           </Top>
 
+          {/* Used for PNG and SVG exports to render message banner to render in output */}
+          {downloadStatus.isLoading && (
```

Comment:
Used for PNG and SVG exports to render message banner to render in output

this will be hidden behind the loading screen and disappear on download complete